frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareCustomPackage/SoftwareCustomPackage.tsx

```diff
@@ -157,6 +157,9 @@ const SoftwareCustomPackage = ({
         )
       );
     } catch (e) {
+      console.log(
+        ">>>>> here it should call the same getErrorMessage as for VPP apps..."
+      );
```

Comment:
This should handle the error similar to how it does in `frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareAppStoreVpp/helpers.tsx`, extracting the error message from the server if it matches and formatting it properly. I didn't do it myself because this requires some refactoring so we can share cleanly between VPP and Custom Package.

Reply:
sgtm, I can get to this today